packages/extensions/openapi-to-cadl/test/bodyComplex/cadl-output/models.cadl

```diff
@@ -4,35 +4,26 @@ using Cadl.Rest;
 
 namespace AutoRestComplexTestService;
 
-enum CMYKColorsKnownValues {
-  Cyan: "cyan",
+enum CMYKColors {
```

Comment:
This will raise a warning in Azure because it isn't PascalCase ("CmykColors").

Reply:
This is the way it is defined in Swagger, if the editor/compiler rise Warnings I think it is okay to let Typespec authors fix it, rather than getting into the business of autogenerating names in the right casing.

Do we also have warnings for model names?

Reply:
Added some pretransform renames to help with these